src/main/rx/crsf.c

```diff
@@ -378,12 +378,10 @@ STATIC_UNIT_TESTED float crsfReadRawRC(const rxRuntimeState_t *rxRuntimeState, u
      * max  2048 -> 2012us
      */
 
-    if (crsfFrame.frame.type == CRSF_FRAMETYPE_RC_CHANNELS_PACKED_FULL_RANGE)
-    {
+    if (crsfFrame.frame.type == CRSF_FRAMETYPE_RC_CHANNELS_PACKED_FULL_RANGE) {
         return (0.5f * (float)crsfChannelData[chan]) + 988;
     }
-    else
-    {
+    else {
```

Comment:
Please remove bracket on line 383 and put it on line 384 like `} else {`

Reply:
Whoops, sorry